src/python/pants/core/goals/tailor.py

```diff
@@ -285,6 +285,36 @@ def register_options(cls, register):
             f"to the one it replaces (see {doc_url('macros')}).",
         )
 
+        register(
+            "--ignore-paths",
+            advanced=True,
+            type=list,
+            member_type=str,
+            help=(
+                "Do not edit or create BUILD files at these paths.\n\n"
```

Comment:
Note that this option name & description are anticipating unifying `update-build-files` and `tailor`. The edits might not be adding targets! Could be fixing deprecations, adding Black, etc.

Further, the paths could be `pants.toml` files in the future.